src/tests/unit/doughs-dao.test.js

```diff
@@ -179,4 +184,52 @@ describe('test doughs dao', () => {
       });
     });
   });
+  context('getDoughById', () => {
+    let result;
+    beforeEach(() => {
+      connectionStub = sinon.stub().returns(executeReturn);
+      doughsDao = proxyquireDoughsDao(
+        connectionStub,
+        serializeDoughsStub,
+        serializeDoughStub,
+      );
+    });
+    context('when called with an integer id', () => {
+      beforeEach(async () => {
+        result = doughsDao.getDoughById(1);
+        await result;
+      });
+
+      it('calls execute with the right query and bind params', () => {
+        connectionStub.should.have.been.calledWith(
+          getDoughByIdQuery,
+          { id: 1 },
+        );
+      });
+
+      it('extracts rows from the return and passes them to the serializer', () => {
+        serializeDoughStub.should.have.been.calledWith(
+          executeReturnRows,
+          'doughs/1',
+        );
+      });
+
+      it('returns the result from the serializer', async () => result
+        .should.eventually.deep.equal(baseGetDoughsReturn));
```

Comment:
I mentioned it on the yesterday code review but I guess you missed it and I didn't catch it on the second round code review lol. Again, data should be an "object" instead of a singleton array if we call get by ID method.

Reply:
Oh yeah, that's a problem. I will look into what's required to fix that.